test/bioprinttest.c

```diff
@@ -11,8 +11,12 @@
 
 #include <stdio.h>
 #include <string.h>
-#include <openssl/opensslconf.h>
-#include OPENSSL_UNISTD
+#include <openssl/e_os2.h>
+#ifdef OPENSSL_SYS_WINDOWS
+# include <winsock.h>
```

Comment:
So if we have configured `no-stdio`, we don't run bioprinttest at all?

I'm pausing for now, but am seriously considering undoing the BIO_printf avoiding stuff in this file.